technology/typemapping/store/cassandra/src/main/java/com/github/mizool/technology/typemapping/store/cassandra/CassandraUuidValueLoadStrategy.java

```diff
@@ -0,0 +1,26 @@
+package com.github.mizool.technology.typemapping.store.cassandra;
+
+import java.time.ZoneId;
+
+import com.datastax.driver.core.Row;
+import com.github.mizool.technology.typemapping.business.DataType;
+
+/**
```

Comment:
I wanted to tell you to remove this comment as it's very majestix specific. However, reading `UUID`s as `String` is the main problem here. I'm wondering if this should indeed return a `UUID` and if we should add `DataType.UUID` to the `enum` and AsciiDoc table.

Since we don't support `UUID`s for JDBC, let's simply remove this class from mizool for now. This keeps mizool consistent. If any application, like majestix, needs this kind of value loading for `UUID`s, it can easily implement this class and the `LoadStrategyResolver`s `Instance` mechanism will add the support.